src/shared/components/gene-search-panel/GeneSearchPanel.module.css

```diff
@@ -94,6 +94,12 @@
   cursor: pointer;
 }
 
+.geneSymbol {
+  font-weight: var(--font-weight-bold);
+  text-transform: uppercase;
```

Comment:
Umm, we shouldn't modify the text of the symbol

Reply:
> XD shows all caps though

Some symbols will have all capital letters; some will have lower-case letters (e.g. genes of C. elegans). This cannot be a design decision; it is the way appropriate scientific communities decide to name a gene.